content/general/concepts/what-is-csv/what-is-csv.md

```diff
@@ -0,0 +1,36 @@
+---
+Title: "What is CSV"
+Subjects:
+  - "Computer Science"
+  - "Data Science"
+Tags:
+  - "Files"
+Catalog Content:
+  - "https://www.codecademy.com/learn/paths/code-foundations"
+  - "https://www.codecademy.com/learn/paths/computer-science"
+---
+
+Comma-Seperated Values (CSV) is a text file format where individual values are seperated by commas. It is normally used to store tabular data and is favored for its simple structure allowing for straightforward parsing and data exchange.
+
+To store tabular data, CSV files usually represent each line as a new row within the table and the values delimited by commas as the cells within each row. However, this practice is not formalized so the interpretation of the cell's contents is normally application-specific. For example, some applications use quotation marks ("") or a backslash (\\) to denote literal characters that allow the cells themselves to also contain the comma character.
```

Comment:
Maybe move "For example, ..." to a new paragraph?

And also single tick the "" and \\. So like:

For example, some applications use quotation marks (`"`) or a backslash (`\`) to denote literal characters that allow the cells themselves to also contain the comma character.